.idea/SumString.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All these files are not needed for this task and review.

Please read more about `.gitignore` file:
https://git-scm.com/docs/gitignore

Reply:
Files deleted